HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -215,12 +215,20 @@ def customiseForOffline(process):
     return process
 
 
+def customizeHLTfor42514(process):
```

Comment:
@missirol @Martin-Grunewald I don't remember: should we put here the number of this PR, or the one in the master (as  implemented here)?

Reply:
In principle the one for the specific release...

Reply:
> In principle the one for the specific release...

True, but I would suggest to keep the number of the original PR. The reason is that keeping the files identical (as much as possible) across cycles helps when cherry-picking, avoiding silly conflicts.

Reply:
@missirol @Martin-Grunewald I assume from the discussion above that we are fine with keeping the old release number. Please shout if not the case.